### ISSUE TITLE
Separate example stories source code

### ISSUE THREAD

**Issue by AXeL-dev** [clauderic/dnd-kit#425]
Hello & thanks for the awesome library,

I just wanted to suggest a code separation for the [example stories](https://5fc05e08a4a65d0021ae0bf2-mdsmdaavqx.chromatic.com) (like literally for each story), & maybe provide the source code of each story on the examples website, because i found that it's really hard & time consuming to read all the logic of [Draggable stories](https://github.com/clauderic/dnd-kit/blob/master/stories/1%20-%20Core/Draggable/1-Draggable.story.tsx) just to understand how a simple dragging example work.

It would be a way easier for beginners on the library to have simple examples with simple source code, so they can easily get used to the library.

Thanks again.

**Comment by clauderic** [comment]
Definitely. I would welcome any PRs to that effect. For now, this isn't something I have time to do, but I definitely agree that this would be super valuable.

**Comment by ThiefMaster** [comment]
+1 on this, right now it's a huge mess to find a simple example :/

**Comment by ABSanthosh** [comment]
@clauderic I'm happy to make PRs. It would be a great starting point if you can atleast document the existing examples a little bit so reverse engineering can be bit less painful.

**Comment by clauderic** [comment]
This issue was opened against a previous version of dnd-kit. The library has since undergone a major rewrite.

It has been rebuilt from the ground up with support for **multiple frameworks** (React, Vue, Svelte, Solid, and vanilla JavaScript), improved performance, and a new API.

- **Documentation**: https://dndkit.com
- **Migration guide**: https://dndkit.com/react/guides/migration
- **GitHub**: https://github.com/clauderic/dnd-kit

If this issue is still relevant to the latest version, please open a new issue.
